lectures/js/containers/renderCourseList.js

```diff
@@ -26,6 +26,6 @@ export function renderCourseList(courseList, filter, pageNumber) {
   if (!paginatedCourseList.length) {
     container.innerHTML = `<p class="course-card-empty-message">조건에 맞는 강의가 없습니다.</p>`;
   } else {
-    container.innerHTML = paginatedCourseList.map(courseCard).join("");
+    container.innerHTML = paginatedCourseList.map(CourseCard).join("");
```

Comment:
```suggestion
    container.innerHTML = paginatedCourseList
      .map((courseItem) => CourseCard(courseItem))
      .join("")
```

이렇게 대신 수정 부탁드립니다!(제 불찰입니다!) 🙏🏻